packages/ui/src/components/va-virtual-scroller/VaVirtualScroller.stories.ts

```diff
@@ -1,12 +1,285 @@
+      v-slot="{item, index}"
+    >
+      <div>{{ index }} - {{ item }}</div>
+    </va-virtual-scroller>
+  `,
+})
+
+Bench.play = async ({ canvasElement, step }) => {
+  const scrollers = document.querySelectorAll('.va-virtual-scroller')
+  const canvas = within(canvasElement)
+  const lists = canvas.getAllByRole('list')
+
+  await step('wrapper height is 200px', async () => {
+    expect((scrollers[0] as HTMLElement).style.height).toEqual('200px')
+    expect((scrollers[1] as HTMLElement).style.height).toEqual('200px')
+  })
+
+  await step('the virtual scroller that has 20 bench is 40 items larger than no bench virual scroller', async () => {
+    expect(lists[1].children.length - lists[0].children.length).toBe(40)
+  })
+}
+
+export const CustomKey = () => ({
+  components: { VaVirtualScroller },
+  data: () => ({ hugeObjectsArray }),
+  template: `
+    <va-virtual-scroller
+      :items="hugeObjectsArray"
+      :bench="10"
+      track-by="value"
+      :wrapper-size="200"
+      v-slot="{item, index}"
+    >
+      <div>{{ index }} - {{ item.value }}</div>
+    </va-virtual-scroller>
+  `,
+})
+
+CustomKey.play = async ({ step }) => {
+  const scroller = document.querySelector('.va-virtual-scroller') as HTMLElement
+
+  await step('wrapper height is 200px', async () => {
+    expect(scroller.style.height).toEqual('200px')
+  })
+}
+
+export const LoopingComponent = () => ({
+  components: { VaVirtualScroller, VaBadge },
+  data: () => ({ hugeObjectsArray }),
+  template: `
+    <va-virtual-scroller
+      :items="hugeObjectsArray"
+      :bench="20"
+      track-by="value1"
+      :wrapper-size="200"
+      v-slot="{item}"
+    >
+      <va-badge :text="\`item \${item.value}\`" color="success" />
+    </va-virtual-scroller>
+  `,
+})
+
+LoopingComponent.play = async ({ step }) => {
+  const scroller = document.querySelector('.va-virtual-scroller') as HTMLElement
+
+  await step('wrapper height is 200px', async () => {
+    expect(scroller.style.height).toEqual('200px')
+  })
 }
 
-export const Default = defineComponent({
+export const DifferentSizesAndMargins = () => ({
+  components: { VaVirtualScroller, VaButton },
+  data: () => ({ hugeObjectsArray }),
+  template: `
+    <va-virtual-scroller
+      :items="hugeObjectsArray"
+      :bench="20"
+      track-by="value"
+      :wrapper-size="200"
+      v-slot="{item, index}"
+    >
+      <div :class="index % 2 ? 'pb-1' : 'pb-6'">
+        <va-button :size="index % 2 ? 'small' : 'medium'">{{ item.value }}</va-button>
+      </div>
+    </va-virtual-scroller>
+  `,
+})
+
+DifferentSizesAndMargins.play = async ({ canvasElement, step }) => {
+  const canvas = within(canvasElement)
+  const list = canvas.findByRole('list')
+  const scroller = document.querySelector('.va-virtual-scroller') as HTMLElement
+
+  await step('wrapper height is 200px', async () => {
+    expect(scroller.style.height).toEqual('200px')
+  })
+}
+
+export const RemWrapperAndItemSizeValue = () => ({
```

Comment:
This story is not great. Instead we need:
* One story for `wrapper-size`
  * You can make it similarly to [this story](https://ui-storybook-develop.vuestic.dev/?path=/story/vaalert--border).
  * Should have: [100] (default, please don't define a prop), [auto], [number px], [number rem].
* One story for `item-size`
  * 0 [default], [number], [number px], [number rem]
* You can select any number you see appropriate. 
* Let's also remove other wrapper size and item-size demos as they become redundant.